package/MDAnalysis/lib/util.py

```diff
@@ -257,6 +257,20 @@ def openany(datasource, mode='r', reset=True):
         stream.close()
 
 
+def _bz2_open_py2(filename, mode):
```

Comment:
Doesn't change much. I found it marginally more readable like that but I can put the def in the except.